pcsx2/CMakeLists.txt

```diff
@@ -1112,6 +1111,7 @@ include_directories(
     gui
     x86
     ${CMAKE_BINARY_DIR}/pcsx2/gui
+    ${CMAKE_SOURCE_DIR}/3rdparty/armips
```

Comment:
Have you tested without this?  If armips did things correctly it shouldn't be needed.

Reply:
Unfortunately, the latest release version (which is what I referenced here) does not do it correctly. I've rewritten the whole CMake project for the master branch though, so everything is good there now. I could update to use that instead, too.

Reply:
That would be my preference

Reply:
Okay. I'll update it to the current master branch. That will require C++17 and bring in a recursive submodule (https://github.com/gulrak/filesystem, as std::filesystem isn't available on all platforms - std::filesystem _can_ be used if it is available though), is that a problem?

Reply:
I've updated it now. The VS solution uses std::filesystem, as Microsoft supports it well. On the CMake side it's off by default, but can be enabled through an option. Most recent Linux distros should have no problem it, but you need quite a recent MacOS version to get native support.